pkg/ps/ps.go

```diff
@@ -301,5 +301,5 @@ func (a SortPSContainers) Swap(i, j int) { a[i], a[j] = a[j], a[i] }
 type SortPSCreateTime struct{ SortPSContainers }
 
 func (a SortPSCreateTime) Less(i, j int) bool {
-	return a.SortPSContainers[i].Created > a.SortPSContainers[j].Created
+	return a.SortPSContainers[i].Created.Before(a.SortPSContainers[j].Created)
```

Comment:
I'm a bit confused.  Was the ">" correct for the original version of this "Less()" function?  If so, does this need to be `After` rather than `Before`, or are you correcting a bad sort?

Back to my tea cup.

Reply:
@TomSweeneyRedHat I'm a mistake in original code see https://github.com/containers/podman/pull/8427/files#diff-6fb97e57a192c9eae49da7e6a2ca7783e8d30eaf4d323cf55b56b28861016eb4L140

Reply:
@jwhonce "I'm a mistake in original code", I wasn't aware you've gone full Tron in this project.  That's a bit scary! :smile: 
Thx for the 411